ui/app/helpers/utils/util.test.js

```diff
@@ -254,5 +254,20 @@ describe('util', function () {
         assert(result)
       })
     })
+
+    describe('#getRandomFileName', function () {
+      it('should only return a string containing alphanumeric characters', function () {
+        const result = util.getRandomFileName()
+        assert(result.match(/^[a-zA-Z0-9]*$/g))
```

Comment:
If we know that this is of a certain size, we could put that here too maybe? Something like so:

```suggestion
        assert(result.match(/^[a-zA-Z0-9]{6,12}$/g))
```

Reply:
Can do that as well, I thought it was more semantic to have one test to do with content and one to do with length

Reply:
No, you're right, this is good as-is. Good point.